fractal-puppy/Engine
# Fixed player attack delay

[Bug #957](https://app.hacknplan.com/p/85404/kanban?categoryId=8&boardId=212215&taskId=957&tabId=basicinfo)

**Before:** Player attack animation didn't sync with damage applied to an enemy.
**Now:** it is sync.

**To test:**
1. Open graveyard scene.
2. Hit enemies and see if the number of the damage appears synchronized with the attack animation.
